cirrus/server/cirrus/main.py

```diff
@@ -16,6 +17,11 @@
 logger = logging.getLogger(__name__)
 
 
+class FeatureRequest(BaseModel):
+    client_id: Optional[str]
+    context: Optional[Dict[str, Any]]
```

Comment:
Should these be `Optional`?

Reply:
okay so if we decide we want to return default config whether key is present or not, then yes it should be optional, if we don't want to continue and required these two keys then we can remove this optional